src/main/java/org/highmed/numportal/service/ehrbase/EhrBaseService.java

```diff
@@ -263,7 +267,7 @@ private List<String> getAndRemoveEhrStatusColumn(QueryResponseData compositions)
     }
     String ehrStatusPath = columns.get(0).get(PATH);
     if (ehrStatusPath == null || !ehrStatusPath.equals("/" + ehrBaseProperties.getIdPath())) {
-      throw new SystemException(EhrBaseService.class, QUERY_RESULT_DOESN_T_CONTAIN_EHR_STATUS_COLUMN);
+      //throw new SystemException(EhrBaseService.class, QUERY_RESULT_DOESN_T_CONTAIN_EHR_STATUS_COLUMN);
```

Comment:
Wieso wird hier die ErrorMessages auskommentiert und nicht direkt entfernt?

Reply:
Thank you for noticing! Only line 45 was meant for this pull request.